.circleci/config.yml

```
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-       - image: circleci/ruby:2.5.1-node-browsers
+       - image: cimg/ruby:3.3-node
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 7

(indentation)